src/commcare_cloud/environment/paths.py

```diff
@@ -49,6 +51,10 @@ def get_env_file_path(self, filename):
     def public_yml(self):
         return self.get_env_file_path('public.yml')
 
+    @lazy_immutable_property
+    def group_vars_all_yml(self):
+        return self.ansible_path / "group_vars/all.yml"
```

Comment:
Is this syntax new? I assume `Path` is overriding `/` to concatenate the parts, but I have never seen this before.

Reply:
`Path` was introduced in Python 3.4. It's really nice.